HISTORY.md

```diff
@@ -1,6 +1,11 @@
 unreleased
 ==========
 
+  * deps: cookie@0.5.0
+    - Add priority option
```

Comment:
The changes we list are only what affect cookie-parser module, and this change does not. I'm going to remove this from the pr, no worries, but just noting in case you wonder why it was removed.

Reply:
I've pushed the change to remove this line:
```
- Add priority option
```

Looking forward to getting this merged if all looks good 🙇🏼

Reply:
This was solved in https://github.com/expressjs/cookie-parser/pull/92/commits/5a0e0cac7c46ef2b9da26bfe8350cc56a329419c